### ISSUE TITLE
2025-04-07, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#133]
# Today's Arxiv Papers

(1) [Do Larger Language Models Imply Better Reasoning? A Pretraining Scaling Law for Reasoning](http://arxiv.org/abs/2504.03635v1), 2025-04-04

(2) [Align to Structure: Aligning Large Language Models with Structural Information](http://arxiv.org/abs/2504.03622v1), 2025-04-04

(3) [VISTA-OCR: Towards generative and interactive end to end OCR models](http://arxiv.org/abs/2504.03621v1), 2025-04-04

(4) [Multilingual Retrieval-Augmented Generation for Knowledge-Intensive Task](http://arxiv.org/abs/2504.03616v1), 2025-04-04

(5) [AIR: A Systematic Analysis of Annotations, Instructions, and Response Pairs in Preference Dataset](http://arxiv.org/abs/2504.03612v1), 2025-04-04

(6) [APIGen-MT: Agentic Pipeline for Multi-Turn Data Generation via Simulated Agent-Human Interplay](http://arxiv.org/abs/2504.03601v1), 2025-04-04

(7) [EnrichIndex: Using LLMs to Enrich Retrieval Indices Offline](http://arxiv.org/abs/2504.03598v1), 2025-04-04

(8) [Hallucination Detection on a Budget: Efficient Bayesian Estimation of Semantic Entropy](http://arxiv.org/abs/2504.03579v1), 2025-04-04

(9) [SynWorld: Virtual Scenario Synthesis for Agentic Action Knowledge Refinement](http://arxiv.org/abs/2504.03561v1), 2025-04-04

(10) [Agentic Knowledgeable Self-awareness](http://arxiv.org/abs/2504.03553v1), 2025-04-04

(11) [Diverse In-Context Example Selection After Decomposing Programs and Aligned Utterances Improves Semantic Parsing](http://arxiv.org/abs/2504.03541v1), 2025-04-04

(12) [Neutralizing the Narrative: AI-Powered Debiasing of Online News Articles](http://arxiv.org/abs/2504.03520v1), 2025-04-04

(13) [Structured Legal Document Generation in India: A Model-Agnostic Wrapper Approach with VidhikDastaavej](http://arxiv.org/abs/2504.03486v1), 2025-04-04

(14) [LLMSched: Uncertainty-Aware Workload Scheduling for Compound LLM Applications](http://arxiv.org/abs/2504.03444v1), 2025-04-04

(15) [Know What You do Not Know: Verbalized Uncertainty Estimation Robustness on Corrupted Images in Vision-Language Models](http://arxiv.org/abs/2504.03440v1), 2025-04-04

(16) [Locations of Characters in Narratives: Andersen and Persuasion Datasets](http://arxiv.org/abs/2504.03434v1), 2025-04-04

(17) [Online Difficulty Filtering for Reasoning Oriented Reinforcement Learning](http://arxiv.org/abs/2504.03380v1), 2025-04-04

(18) [Sustainable LLM Inference for Edge AI: Evaluating Quantized LLMs for Energy Efficiency, Output Accuracy, and Inference Latency](http://arxiv.org/abs/2504.03360v1), 2025-04-04

(19) [Detecting Stereotypes and Anti-stereotypes the Correct Way Using Social Psychological Underpinnings](http://arxiv.org/abs/2504.03352v1), 2025-04-04

(20) [Talk2X -- An Open-Source Toolkit Facilitating Deployment of LLM-Powered Chatbots on the Web](http://arxiv.org/abs/2504.03343v1), 2025-04-04

(21) [BabyLM's First Words: Word Segmentation as a Phonological Probing Task](http://arxiv.org/abs/2504.03338v1), 2025-04-04

(22) [Evaluating Compact LLMs for Zero-Shot Iberian Language Tasks on End-User Devices](http://arxiv.org/abs/2504.03312v1), 2025-04-04

(23) [Noise Augmented Fine Tuning for Mitigating Hallucinations in Large Language Models](http://arxiv.org/abs/2504.03302v1), 2025-04-04

(24) [Stance-Driven Multimodal Controlled Statement Generation: New Dataset and Task](http://arxiv.org/abs/2504.03295v1), 2025-04-04

(25) [Towards Effective EU E-Participation: The Development of AskThePublic](http://arxiv.org/abs/2504.03287v1), 2025-04-04

(26) [Do Large Language Models Solve the Problems of Agent-Based Modeling? A Critical Review of Generative Social Simulations](http://arxiv.org/abs/2504.03274v1), 2025-04-04

(27) [Inherent and emergent liability issues in LLM-based agentic systems: a principal-agent perspective](http://arxiv.org/abs/2504.03255v1), 2025-04-04

(28) [Augmenting Human Cognition With Generative AI: Lessons From AI-Assisted Decision-Making](http://arxiv.org/abs/2504.03207v1), 2025-04-04

(29) [Enhancing Personalized Multi-Turn Dialogue with Curiosity Reward](http://arxiv.org/abs/2504.03206v1), 2025-04-04

(30) [Explain with Visual Keypoints Like a Real Mentor! A Benchmark for Multimodal Solution Explanation](http://arxiv.org/abs/2504.03197v1), 2025-04-04

(31) [Learning Natural Language Constraints for Safe Reinforcement Learning of Language Agents](http://arxiv.org/abs/2504.03185v1), 2025-04-04

(32) [Multi-lingual Multi-turn Automated Red Teaming for LLMs](http://arxiv.org/abs/2504.03174v1), 2025-04-04

(33) [Efficient Dynamic Clustering-Based Document Compression for Retrieval-Augmented-Generation](http://arxiv.org/abs/2504.03165v1), 2025-04-04

(34) [DeepResearcher: Scaling Deep Research via Reinforcement Learning in Real-world Environments](http://arxiv.org/abs/2504.03160v1), 2025-04-04

(35) [Beyond the Next Token: Towards Prompt-Robust Zero-Shot Classification via Efficient Multi-Token Prediction](http://arxiv.org/abs/2504.03159v1), 2025-04-04

(36) [Synthesizing Optimal Object Selection Predicates for Image Editing using Lattices](http://arxiv.org/abs/2504.03155v1), 2025-04-04

(37) [TokenFLEX: Unified VLM Training for Flexible Visual Tokens Inference](http://arxiv.org/abs/2504.03154v1), 2025-04-04

(38) [Why Reasoning Matters? A Survey of Advancements in Multimodal Reasoning (v1)](http://arxiv.org/abs/2504.03151v1), 2025-04-04

(39) [A Human Digital Twin Architecture for Knowledge-based Interactions and Context-Aware Conversations](http://dx.doi.org/10.17605/OSF.IO/KEH9T), 2025-04-04

(40) [LightPROF: A Lightweight Reasoning Framework for Large Language Model on Knowledge Graph](http://arxiv.org/abs/2504.03137v1), 2025-04-04

(41) [Les Dissonances: Cross-Tool Harvesting and Polluting in Multi-Tool Empowered LLM Agents](http://arxiv.org/abs/2504.03111v1), 2025-04-04

(42) [Single-Pass Document Scanning for Question Answering](http://arxiv.org/abs/2504.03101v1), 2025-04-04

(43) [Unlocking the AMD Neural Processing Unit for ML Training on the Client Using Bare-Metal-Programming Tools](http://arxiv.org/abs/2504.03083v1), 2025-04-03

(44) [AD-GPT: Large Language Models in Alzheimer's Disease](http://arxiv.org/abs/2504.03071v1), 2025-04-03

(45) [Design of AI-Powered Tool for Self-Regulation Support in Programming Education](http://arxiv.org/abs/2504.03068v1), 2025-04-03

(46) [Task as Context Prompting for Accurate Medical Symptom Coding Using Large Language Models](http://dx.doi.org/10.1145/3721201.3721383), 2025-04-03

(47) [LLM Library Learning Fails: A LEGO-Prover Case Study](http://arxiv.org/abs/2504.03048v1), 2025-04-03

(48) [Extending CREAMT: Leveraging Large Language Models for Literary Translation Post-Editing](http://arxiv.org/abs/2504.03045v1), 2025-04-03

(49) [Ontologies in Design: How Imagining a Tree Reveals Possibilites and Assumptions in Large Language Models](http://dx.doi.org/10.1145/3706598.3713633), 2025-04-03

(50) [AuDeRe: Automated Strategy Decision and Realization in Robot Planning and Control via LLMs](http://arxiv.org/abs/2504.03015v1), 2025-04-03

(51) [What People Share With a Robot When Feeling Lonely and Stressed and How It Helps Over Time](http://arxiv.org/abs/2504.02991v1), 2025-04-03

(52) [Language Models Guidance with Multi-Aspect-Cueing: A Case Study for Competitor Analysis](http://arxiv.org/abs/2504.02984v1), 2025-04-03

(53) [Hummus: A Dataset of Humorous Multimodal Metaphor Use](http://arxiv.org/abs/2504.02983v1), 2025-04-03

(54) [CoLa -- Learning to Interactively Collaborate with Large LMs](http://arxiv.org/abs/2504.02965v1), 2025-04-03

(55) [Digital Forensics in the Age of Large Language Models](http://arxiv.org/abs/2504.02963v1), 2025-04-03

(56) [Cultural Learning-Based Culture Adaptation of Language Models](http://arxiv.org/abs/2504.02953v1), 2025-04-03

(57) [VARGPT-v1.1: Improve Visual Autoregressive Large Unified Model via Iterative Instruction Tuning and Reinforcement Learning](http://arxiv.org/abs/2504.02949v1), 2025-04-03

(58) [Sparse Autoencoders Learn Monosemantic Features in Vision-Language Models](http://arxiv.org/abs/2504.02821v1), 2025-04-03

(59) [Generative Evaluation of Complex Reasoning in Large Language Models](http://arxiv.org/abs/2504.02810v1), 2025-04-03

(60) [MegaMath: Pushing the Limits of Open Math Corpora](http://arxiv.org/abs/2504.02807v1), 2025-04-03

(61) [A Survey of Large Language Models in Mental Health Disorder Detection on Social Media](http://arxiv.org/abs/2504.02800v2), 2025-04-03

(62) [A Framework for Robust Cognitive Evaluation of LLMs](http://arxiv.org/abs/2504.02789v1), 2025-04-03

(63) [From Consumption to Collaboration: Measuring Interaction Patterns to Augment Human Cognition in Open-Ended Tasks](http://arxiv.org/abs/2504.02780v1), 2025-04-03

(64) [BT-ACTION: A Test-Driven Approach for Modular Understanding of User Instruction Leveraging Behaviour Trees and LLMs](http://arxiv.org/abs/2504.02779v1), 2025-04-03

(65) [HyperRAG: Enhancing Quality-Efficiency Tradeoffs in Retrieval-Augmented Generation with Reranker KV-Cache Reuse](http://arxiv.org/abs/2504.02921v1), 2025-04-03

(66) [MultiBLiMP 1.0: A Massively Multilingual Benchmark of Linguistic Minimal Pairs](http://arxiv.org/abs/2504.02768v1), 2025-04-03

(67) [How Deep Do Large Language Models Internalize Scientific Literature and Citation Practices?](http://arxiv.org/abs/2504.02767v1), 2025-04-03

(68) [Enhancing LLM Robustness to Perturbed Instructions: An Empirical Study](http://arxiv.org/abs/2504.02733v1), 2025-04-03

(69) [Why do LLMs attend to the first token?](http://arxiv.org/abs/2504.02732v2), 2025-04-03

(70) [ERPO: Advancing Safety Alignment via Ex-Ante Reasoning Preference Optimization](http://arxiv.org/abs/2504.02725v1), 2025-04-03

(71) [TeleMoM: Consensus-Driven Telecom Intelligence via Mixture of Models](http://arxiv.org/abs/2504.02712v1), 2025-04-03

(72) [The Hidden Space of Safety: Understanding Preference-Tuned LLMs in Multilingual context](http://arxiv.org/abs/2504.02708v1), 2025-04-03

(73) [LLM for Complex Reasoning Task: An Exploratory Study in Fermi Problems](http://arxiv.org/abs/2504.02671v1), 2025-04-03

(74) [Affordable AI Assistants with Knowledge Graph of Thoughts](http://arxiv.org/abs/2504.02670v1), 2025-04-03

(75) [Prompt Optimization with Logged Bandit Data](http://arxiv.org/abs/2504.02646v1), 2025-04-03

(76) [EmbodiedSense: Understanding Embodied Activities with Earphones](http://arxiv.org/abs/2504.02624v1), 2025-04-03

(77) [Multi-Mission Tool Bench: Assessing the Robustness of LLM based Agents through Related and Dynamic Missions](http://arxiv.org/abs/2504.02623v1), 2025-04-03

(78) [Exploring undercurrents of learning tensions in an LLM-enhanced landscape: A student-centered qualitative perspective on LLM vs Search](http://arxiv.org/abs/2504.02622v1), 2025-04-03

(79) [Multi-SWE-bench: A Multilingual Benchmark for Issue Resolving](http://arxiv.org/abs/2504.02605v1), 2025-04-03

(80) [LexPam: Legal Procedure Awareness-Guided Mathematical Reasoning](http://arxiv.org/abs/2504.02590v1), 2025-04-03

(81) [Rethinking RL Scaling for Vision Language Models: A Transparent, From-Scratch Framework and Comprehensive Evaluation Scheme](http://arxiv.org/abs/2504.02587v2), 2025-04-03

(82) [Language Models reach higher Agreement than Humans in Historical Interpretation](http://arxiv.org/abs/2504.02572v1), 2025-04-03

(83) [Bias in Large Language Models Across Clinical Applications: A Systematic Review](http://arxiv.org/abs/2504.02917v1), 2025-04-03

(84) [Leveraging LLM For Synchronizing Information Across Multilingual Tables](http://arxiv.org/abs/2504.02559v1), 2025-04-03

(85) [Exploring Individual Factors in the Adoption of LLMs for Specific Software Engineering Tasks](http://arxiv.org/abs/2504.02553v1), 2025-04-03

(86) [GPG: A Simple and Strong Reinforcement Learning Baseline for Model Reasoning](http://arxiv.org/abs/2504.02546v1), 2025-04-03

(87) [UNDO: Understanding Distillation as Optimization](http://arxiv.org/abs/2504.02521v1), 2025-04-03

(88) [A Memory-Augmented LLM-Driven Method for Autonomous Merging of 3D Printing Work Orders](http://arxiv.org/abs/2504.02509v1), 2025-04-03

(89) [ZClip: Adaptive Spike Mitigation for LLM Pre-Training](http://arxiv.org/abs/2504.02507v1), 2025-04-03

(90) [Inference-Time Scaling for Generalist Reward Modeling](http://arxiv.org/abs/2504.02495v1), 2025-04-03

(91) [Noiser: Bounded Input Perturbations for Attributing Large Language Models](http://arxiv.org/abs/2504.02911v1), 2025-04-03

(92) [MG-MotionLLM: A Unified Framework for Motion Comprehension and Generation across Multiple Granularities](http://arxiv.org/abs/2504.02478v1), 2025-04-03

(93) [Multimodal Fusion and Vision-Language Models: A Survey for Robot Vision](http://arxiv.org/abs/2504.02477v1), 2025-04-03

(94) [Retrieval-Augmented Purifier for Robust LLM-Empowered Recommendation](http://arxiv.org/abs/2504.02458v1), 2025-04-03

(95) [Cognitive Memory in Large Language Models](http://arxiv.org/abs/2504.02441v1), 2025-04-03

(96) [A Multi-Level Sentiment Analysis Framework for Financial Texts](http://arxiv.org/abs/2504.02429v1), 2025-04-03

(97) [Narrative Studio: Visual narrative exploration using LLMs and Monte Carlo Tree Search](http://arxiv.org/abs/2504.02426v1), 2025-04-03

(98) [Adapting Large Language Models for Multi-Domain Retrieval-Augmented-Generation](http://arxiv.org/abs/2504.02411v1), 2025-04-03

(99) [AnesBench: Multi-Dimensional Evaluation of LLM Reasoning in Anesthesiology](http://arxiv.org/abs/2504.02404v1), 2025-04-03

(100) [DaKultur: Evaluating the Cultural Awareness of Language Models for Danish with Native Speakers](http://arxiv.org/abs/2504.02403v1), 2025-04-03
